### PR TITLE
Change readme for 'Format code' (new extension PHP CS Fixer) with demo png and gif

### PR DESCRIPTION
**Pull request by welkie**
Since "phpcs" does not provide the Format Document feature, recommending "PHP CS Fixer" for this and keeping phpcs recommendation for linting warnings. Added demo png for phpcs and demo gif for PHP CS Fixer.

See issue #172

### CODE REVIEW

**Comment by welkie** [comment]
Understandable. Changed to remove GIF for 3rd party extension, just link now.

File: README.md [hunk 1]
```diff
@@ -40,9 +40,13 @@ I recommend to disable VS Code's built-in PHP IntelliSense by setting `php.sugge
 ### Column-accurate error reporting
 ![Error reporting demo](images/publishDiagnostics.png)
 
-### Format code
+### Code style linting
 Please use a dedicated extension like [PHP CodeSniffer](https://marketplace.visualstudio.com/items?itemName=ikappas.phpcs).
 
+![Code style linting demo](images/phpcs.png)
```
Comment (felixfbecker) [inline README.md]: Please remove this too

**Comment by welkie** [comment]
Changes made.

**Pull request merged**